src/components/payment/Payment.test.js

```diff
@@ -170,6 +178,44 @@ describe('<PaymentSection />', () => {
       })
     })
 
+    it('should trigger an email subscribe if meetup', async () => {
```

Comment:
@ric9176 what about "should trigger an email subscribe if meetup is true and the subscribe to newsletter checkbox is checked"